OneSignalSDK/onesignal/src/main/java/com/onesignal/OSFocusHandler.kt

```diff
@@ -42,21 +42,34 @@ class OSFocusHandler {
 
     fun startOnFocusWork() {
         resetBackgroundState()
+        OneSignal.onesignalLog(
+            OneSignal.LOG_LEVEL.DEBUG,
+            "OSFocusHandler running onAppFocus"
+        )
         OneSignal.onAppFocus()
     }
 
     fun startOnStartFocusWork() {
         if (stopped) {
             stopped = false
-            OSTimeoutHandler.getTimeoutHandler().destroyTimeout(stopRunnable)
             stopRunnable = null
+            OneSignal.onesignalLog(
+                OneSignal.LOG_LEVEL.DEBUG,
+                "OSFocusHandler running onAppStartFocusLogic"
+            )
             OneSignal.onAppStartFocusLogic()
+        } else {
+            resetStopState()
         }
     }
 
     fun startOnStopFocusWork() {
         stopRunnable = Runnable {
             stopped = true
+            OneSignal.onesignalLog(
+                OneSignal.LOG_LEVEL.DEBUG,
+                "OSFocusHandler running onAppStopFocusLogic"
```

Comment:
Small thing, can this log statement be reworded? 
It will seem like we are calling a function named `onAppStopFocusLogic` but it doesn't seem so.

Reply:
NIce catch!